my_site/blog/apps.py

```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
+
+
+class BlogConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
```

Comment:
`default_auto_field` has already defined in `settings.py`. Do we really need to define it in the app too?